Command/UpdateUserHandler.php

```diff
@@ -27,13 +27,17 @@ public function __construct(UserRepositoryCollection $userRepositoryCollection)
      */
     public function handle(UpdateUser $command)
     {
-        $updatingUser = $command->getUser();
-        $user = new User(
-            $command->getUsername(),
-            $command->getPassword(),
-            $command->getDisplayName()
-        );
-        $repository = $this->getUserRepositoryForUser($this->userRepositoryCollection, $updatingUser);
-        $repository->update($updatingUser, $user);
+        $user = $command->getUser();
+
+        $username = $command->getUsername();
```

Comment:
waarom steek je dit allemaal in variabelen die je toch maar 1x gebruikt ?
Je zou dit toch rechtstreeks in het aanmaken van de user kunnen gebruiken ?